plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -90,6 +90,7 @@
     private int maxConcurrentFileRenames = 20;
     private int maxConcurrentMetastoreDrops = 20;
     private int maxConcurrentMetastoreUpdates = 20;
+    private int maxConcurrentMetastoreGetRequests = 40;
```

Comment:
Q: can we consolidate all these max concurrent metastore properties into since property like `hive.max-concurrent-metastore-requests`?